data/topic-schema/src/main/resources/net/corda/schema/P2P.yaml

```diff
@@ -3,6 +3,7 @@ topics:
     name: p2p.in
     consumers:
       - flow
```

Comment:
is the FlowP2PFilterService in flow mapper worker or flow worker now? i think it should probably go into flow mapper worker and then flow is no longer a consumer of the p2p.in